drivers/display/display_sdl.c

```diff
@@ -460,6 +460,7 @@ static int sdl_display_clear(const struct device *dev)
 		break;
 	default:
 		LOG_ERR("Pixel format not supported");
+		return -EINVAL;
```

Comment:
The default branch is never going to be executed so I think it would be better to have code/comments that better reflect that. Maybe at a minimum turn the LOG statement into an assert?

Another option, and possibly better, would be to use if/else ifs like in several other functions in this file.